teamsupport/services.py

```diff
@@ -40,18 +44,77 @@ def __init__(self, org_id, auth_token, **kwargs):
         self.org_id = org_id
         self.auth_token = auth_token
 
-    def get_tickets(self, query_params=None):
+    def search_tickets(self, query_params=None):
+        if query_params is None:
+            query_params = {}
+        query_params['TicketTypeID'] = TICKET_TYPE_SUPPORT
+
         response = self.get('Tickets/', params=query_params)
         content = self.parse_xml_response(response)
         return content
 
-    def create_ticket(self, data):
+    def search_contacts(self, **query_params):
+        response = self.get('Contacts/', params=query_params)
+        return self.parse_xml_response(response)
+
+    def create_contact(self, **query_params):
         response = self.post(
-            'Tickets/', root='Ticket', data=data, send_as_xml=True)
+            'Contacts/', root='Contact', data=query_params, send_as_xml=True)
         return self.parse_xml_response(response)
 
+    def delete_contact(self, contact_id):
+        self.delete('Contacts/{}'.format(contact_id))
+
+    def create_ticket(self, first_name, email, category, name, description):
```

Comment:
I think service methods should remain simple.  And we should glue needed functionality in `models.py`